django_extensions/management/commands/runserver_plus.py

```diff
@@ -73,7 +73,7 @@
 DEFAULT_POLLER_RELOADER_TYPE = getattr(settings, 'RUNSERVERPLUS_POLLER_RELOADER_TYPE', 'auto')
 
 logger = logging.getLogger(__name__)
-_error_files = set()  # type: Set[str]
+_error_files: Set = set()
```

Comment:
I'm curious as to why you dropped the `str`? Was it not correct? Like this, I mean:
```suggestion
_error_files: Set[str] = set()
```

Reply:
Good catch, wasn't intended.